src/media-service.android.ts

```diff
@@ -629,11 +638,21 @@ export namespace dk {
        * Use this function to check, if a URL is allowed.
        */
       private checkUrlAllowed(url: string) {
+        if (android.os.Build.VERSION.SDK_INT < 23) {
+          return true;
+        }
+
+        if (android.os.Build.VERSION.SDK_INT === 23) {
+          // https://developer.android.com/reference/android/security/NetworkSecurityPolicy.html#isCleartextTrafficPermitted()
+          return !!android.security.NetworkSecurityPolicy.getInstance().isCleartextTrafficPermitted();
```

Comment:
I think you wanna move this below the `clearTextHostname` check or no URLs will be allowed on API 23 unless `isCleartextTrafficPermitted` returns true.